templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
The TargetGroup thing is part of the "ApplicationLoadBalancer" system.  Only supports HTTP and HTTPS.  We need TCP as we want to terminate the HTTP connection at the far end and we want to make sure client certs make it through.

My gut is that we'll have to do `aws elb register-instances-with-load-balancer` in the startup script.  The Role should be there (or we can add it to the IAM doc) so it should be a one liner.